inference-engine/src/vpu/myriad_plugin/myriad_executable_network.h

```diff
@@ -35,17 +35,20 @@ class ExecutableNetwork : public InferenceEngine::ExecutableNetworkThreadSafeDef
     explicit ExecutableNetwork(InferenceEngine::ICNNNetwork &network,
                                std::shared_ptr<IMvnc> mvnc,
                                std::vector<DevicePtr> &devicePool,
-                               const MyriadConfig& config);
+                               const MyriadConfig& config,
+                               const ie::ICore* core);
 
     explicit ExecutableNetwork(std::istream& strm,
                                std::shared_ptr<IMvnc> mvnc,
                                std::vector<DevicePtr> &devicePool,
-                               const MyriadConfig& config);
+                               const MyriadConfig& config,
+                               const ie::ICore* core);
```

Comment:
`ExecutableNetwork` has a pointer to a plugin `_plugin`, you can extract `ICore` interface via `_plugin->GetCore()` inside the executable network.